app/src/main/res/values/strings.xml

```diff
@@ -173,7 +176,8 @@
     <string name="govt_advisory_tile">Govt Advisory</string>
     <string name="symptom_tracker_tile">Symptom Tracker</string>
     <string name="contact_tracer_tile">Contact Tracer</string>
-    <string name="chatbot_tile">Chatbots</string>
```

Comment:
No need to remove an unused resource. We will refactor the app before release so such unused variables will go away

Reply:
Okay